scripts/updatechangelog.sh

```diff
@@ -0,0 +1,11 @@
+output="Please refer to each API's \`CHANGELOG.md\` file under the \`packages/\` directory\n\nChangelogs\n-----\n"
+
+SCRIPT_ROOT=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+PACKAGES_URL=https://github.com/googleapis/google-cloud-python/tree/main/packages
+
+for package in $SCRIPT_ROOT/../packages/*/ ; do
+    package_dir=$(basename $package)
+    output+="[$package_dir]("$PACKAGES_URL/$package_dir"/CHANGELOG.md)\n"    
```

Comment:
I think it would be helpful if we could increase list the current version number for each package:
` [$package_dir $package_version]($changelog_url)`

Not a blocker, and certainly we could do this with a chaser PR, but it might be simple enough to do now and just have it done.

Reply:
Done in 1ca3269